src/entities/Author.entity.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  Entity,
+  PrimaryGeneratedColumn,
+  Column,
+  OneToMany,
+  BaseEntity,
+} from 'typeorm';
+import { Song } from './Song.entity';
+import { Album } from './Album.entity';
+
+@Entity()
+export class Author extends BaseEntity {
+  @PrimaryGeneratedColumn()
+  id!: number;
```

Comment:
```suggestion
  id: number;
```
tương tự các field bên dưới nhé, do có method constructor rồi nha